src/RadioGroup.js

```diff
@@ -12,7 +12,7 @@ class RadioGroup extends React.Component {
         }),
         container: PropTypes.string,
         name: PropTypes.string.isRequired,
-        onChange: PropTypes.func.isRequired,
+        onChange: PropTypes.func,
```

Comment:
Another example is if the user is looking at `onClick` instead of `onChange`. Or `onTouchStart`/`onTouchEnd`

Assumptions make an opinionated framework and, if I wanted that, I'd use ember or angular